lib/elastic_apm/span/context.rb

```diff
@@ -28,12 +28,28 @@ def initialize(instance: nil, statement: nil, type: nil, user: nil)
       # @api private
       class Http
         def initialize(url: nil, status_code: nil, method: nil)
-          @url = url
+          @url = sanitize_url(url)
           @status_code = status_code
           @method = method
         end
 
         attr_accessor :url, :status_code, :method
+
+        private
+
+        def sanitize_url(url)
+          uri = URI(url)
+
+          return url unless uri.userinfo
+
+          format(
+            '%s://%s@%s%s',
+            uri.scheme,
+            uri.userinfo.split(':')[0], # username, no password
```

Comment:
You can just call ` uri.user` I believe.

Reply:
You are right!